### DISCUSSION TITLE
Blocage période de réservation

### DISCUSSION THREAD

**Discussion by acdct** [JeromeDevome/GRR#266] [Q&A] [answered]
Bonjour à la communauté,

Afin d'éviter que les usagers puisse réserver à l'avance X années (et ne pas être présents), j'aimerai bloquer le calendrier de réservation et ne permettre la réservation que pendant une période de 6 mois à partir de la date du jour.

Je ne vois malheureusement pas cette possibilité, la seule que je vois est le "début et fin de réservations" dans la configuration, mais cela oblige les administrateur à revenir sur cette date pour la modifier à chaque période.

Savez-vous s'il y a un moyen de réaliser ce que je souhaite faire ?

Merci d'avance pour votre aide.

**Comment by ynaessens** [comment, marked as answer]
Bonjour,
cette fonctionnalité existe : c'est un des paramètres des ressources, accessible en page de configuration de la ressource.
Si vous voulez le faire par lot en utilisant un outil tel que phpmyadmin ou en commande SQL, c'est le champ delais_max_resa_room de la table grr_room. La valeur -1 indique qu'il n'y a pas de limite, sinon une valeur strictement positive indique le nombre de jours au delà duquel il est impossible de réserver.
Cordialement,
YN

**Reply by acdct** [reply]
Top, merci beaucoup YN, je n'avais pas vu cette option.
Bonne journée !
